### PR TITLE
cargo stringtable german [WIP]

### PR DESCRIPTION
**Pull request by shukari**
**When merged this pull request will:**
- some german stringtable entries (September 21, 2017 o.O)

INFO: ok know it shows a conflict will fix that

### CODE REVIEW

File: addons/cargo/stringtable.xml [hunk 1]
```diff
@@ -299,6 +302,7 @@
         </Key>
         <Key ID="STR_ACE_Cargo_size_edenDesc">
             <English>The cargo space required to hold this object (-1 for unloadable)</English>
+            <German>Frachtgröße zum Einladen dieses Objektes (-1 nicht einladbar)</German>
```
Comment (dedmen) [inline addons/cargo/stringtable.xml]: the `required` part is missing there.
Maybe..
`Benötigter Frachtraum zum...`?

File: addons/cargo/stringtable.xml [hunk 2]
```diff
@@ -329,6 +333,7 @@
         </Key>
         <Key ID="STR_ACE_Cargo_paradropTimeCoefficent">
             <English>Paradrop Time Coffecient</English>
+            <German>Türlast Zeitkoeffizient</German>
```
Comment (dedmen) [inline addons/cargo/stringtable.xml]: "Türlast"? Never heard of that and no google results. But I see that's already used in `STR_ACE_Cargo_paradropButton`

Comment (bux) [reply addons/cargo/stringtable.xml]: Ausdruck wird in der BW verwendet.

Comment (Blutze) [reply addons/cargo/stringtable.xml]: Das is entschieden länger als der längste Text in der englischen Übersetzung (welcher da wäre der erste Eintrag in Nametags): turlastabsetzungsdaueranpassung
Glaub "Türlastabsetzungsdaueranpassung" wär eh besseres Deutsch.


Und `<English>Paradrop Time Coffecient</English>` müsste `<English>Paradrop Time Coefficient</English>` sein.

Comment (commy2) [reply addons/cargo/stringtable.xml]: The correct translation for "paradrop" is "Fallschirmabwurf" and not "Türlast".
Terms used in the Bundeswehr are very bad for a translation like this. They're not common use in language. This is a German translation and not a Bundeswehrdeutsch translation.

Comment (commy2) [reply addons/cargo/stringtable.xml]: "Fallschirmabwurf-Zeitkoeffizient"

Comment (shukari) [reply addons/cargo/stringtable.xml]: Im ok with Fallschirmabwurf.

@dedman my google find thousands of results for Türlast in conection with Flugzeug ^^

Comment (dedmen) [reply addons/cargo/stringtable.xml]: dunno who you mentioned there but not me :D 
Okey yeah with Flugzeug I have 51 results. Including one where a guy talks about ACE and that  `Türlast` is a bad translation :D

File: addons/cargo/stringtable.xml (hunk 1 above)
Comment (xetra11) [reply addons/cargo/stringtable.xml]: "Benötigter Frachtraum um dieses Objekt einzuladen (-1 nicht einladbar)"

*clever clogs* 🥇

File: addons/cargo/stringtable.xml (hunk 2 above)
Comment (xetra11) [reply addons/cargo/stringtable.xml]: "Tuerlast" is the right term for dropped cargo out of a plane for the german army. We used this term when we did paradrops and our heavy equipment needed to be dropped separately.

Comment (commy2) [reply addons/cargo/stringtable.xml]: >"Tuerlast" is the right term for dropped cargo out of a plane for the german army.
See: https://github.com/acemod/ACE3/pull/6466#discussion_r204638133

Comment (xetra11) [reply addons/cargo/stringtable.xml]: The english translation uses army terms as well. So where is the problem doing the same in german?

Comment (commy2) [reply addons/cargo/stringtable.xml]: The English version of this string is: "Paradrop Time Coffecient", which is, while very technical, standard English. If there is a bad (=not day to day English) translation anywhere, then it should be changed as well.

>This is a military simulation after all.

And it is also a video game used by civilians. A tooltip is there to be understood by everyone.

Comment (bux) [reply addons/cargo/stringtable.xml]: @commy2 I disagree

Comment (commy2) [reply addons/cargo/stringtable.xml]: I'd be fine with the term being used, and defined in an extra sentence, even if that were a feature of the German string only. Tooltips can be quite lengthy and still fit. Aside from that, I rest my case.

**Pull request closed without merging**